pkg/run/seaweed.go

```diff
@@ -0,0 +1,110 @@
+// Copyright Nitric Pty Ltd.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package run
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/pkg/errors"
+
+	// "github.com/nitrictech/minio/cmd"
+	"github.com/seaweedfs/seaweedfs/weed/command"
+	flag "github.com/seaweedfs/seaweedfs/weed/util/fla9"
+
+	"github.com/nitrictech/cli/pkg/utils"
+)
+
+type SeaweedServer struct {
+	apiPort int // external API port from the seaweedfs s3 API gateway
+}
+
+const (
+	bucketsDir = "./.nitric/buckets"
+	logsDir    = "/.nitric/logs"
+	runPerm    = os.ModePerm // NOTE: octal notation is important here!!!
+)
+
+// Start - Start the local Minio server
```

Comment:
nit: there is a lot of name confusion between minio and seaweed.. could just s/minio/seaweed?

Reply:
fixed